### DISCUSSION TITLE
Creating dynamic diagrams for interacting

### DISCUSSION THREAD

**Discussion by foxfire881** [jgraph/drawio#4405] [Ideas]
Currently, draw.io only supports creating static diagrams, and users cannot interact with them. I hope that draw.io can add functionality for creating dynamic diagrams that users can interact with. For example, it would be great if we could add tooltips to certain components. However, displaying lengthy tooltips directly might take up too much space and not look aesthetically pleasing. Ideally, tooltips could be collapsed by default, and users could expand them when needed by hovering over a component or clicking on it

**Comment by ZsoltMONOLITE** [comment]
Hi, try properties and tooltip plugin
